src/core/utils/getResponse.ts

```diff
@@ -16,16 +16,22 @@ export interface ResponseResolutionContext {
 /**
  * Returns a mocked response for a given request using following request handlers.
  */
-export const getResponse = async <Handler extends Array<RequestHandler>>(
-  request: Request,
-  handlers: Handler,
-  resolutionContext?: ResponseResolutionContext,
-): Promise<ResponseLookupResult | null> => {
+export const getResponse = async <Handler extends Array<RequestHandler>>({
+  request,
+  requestId,
+  handlers,
+  resolutionContext,
+}: {
+  request: Request
+  requestId: string
```

Comment:
The only thing I'm not sure about here is whether requiring `requestId` for the `.run()` method is right. 

While this is not widely used, you can call a request handler directly using its `.run()` method. Just accepting `request` makes sense. Demanding `requestId` means the consumer has to contrive that ID. Hmm.

I think we can make this required and suggest using `crypto.randomUUID()` as a placeholder value if someone has to call a handler directly. If you want to—you must provide a request ID.